### PR TITLE
Seed database.

### PR DESCRIPTION
**Pull request by prettyirrelevant**
## Summary
- Fix bug in database schema.
- Updated config to contain `DATABASE_URL`
- Ran migration.
- Script which populates the database with nft information.

### CODE REVIEW

File: src/utils/misc.ts
Comment (ScMofeoluwa) [inline src/utils/misc.ts:1]: I'd advice you name this file well

File: src/server.ts
Comment (ScMofeoluwa) [inline src/server.ts:1]: prisma automatically connects to your db and you don't necessarily have to call the connect method

File: src/utils/xrpl-client.ts
Comment (ScMofeoluwa) [inline src/utils/xrpl-client.ts:1]: there's no consistency in the function syntax as you're using arrow functions in some files and then normal functions in one. I'd advice you stick with normal functions (personal preference)

File: src/utils/misc.ts
Comment (prettyirrelevant) [reply src/utils/misc.ts:1]: Currently, the module/file serves as a dump for all utility functions that cannot be grouped into a meaninful unit atm.

**Pull request merged**